src/wasm-tree-sitter-language-mode.js

```diff
@@ -1251,45 +1272,49 @@ class WASMTreeSitterLanguageMode {
   shouldUseAsyncIndent() {
     let result = true;
     if (!this.useAsyncParsing || !this.useAsyncIndent) result = false;
+    // If `currentTransactionReparseBudgetMs` somehow isn’t initialized yet, we
+    // can initialize it here for this transaction.
+    this.currentTransactionReparseBudgetMs ??= this.transactionReparseBudgetMs;
     if (this.currentTransactionReparseBudgetMs > 0) {
       result = false;
     }
     return result;
   }
 
-  // Get the suggested indentation level for an existing line in the buffer.
+  // Essential: Get the suggested indentation level for an existing line in the
+  // buffer.
   //
-  // * `bufferRow` - A {Number} indicating the buffer row.
-  // * `tabLength` - A {Number} signifying the length of a tab, in spaces,
+  // * `bufferRow` A {Number} indicating the buffer row.
+  // * `tabLength` A {Number} signifying the length of a tab, in spaces,
   //   according to the current settings of the buffer.
-  // * `options` (optional) - An {Object} will the following properties,all of
-  //   which are optional:
-  // * `options.skipBlankLines`: {Boolean} indicating whether to ignore blank
-  //   lines when determining which row to use as a reference row. Default is
-  //   `true`. Irrelevant if `options.comparisonRow` is specified.
-  // * `options.skipDedentCheck`: {Boolean} indicationg whether to skip the
-  //   second phase of the check and determine only if the current row should
-  //   _start out_ indented from the reference row.
-  // * `options.preserveLeadingWhitespace`: {Boolean} indicating whether to
-  //   adjust the returned number to account for the indentation level of any
-  //   whitespace that may already be on the row. Defaults to `false`.
-  // * `options.forceTreeParse`: {Boolean} indicating whether to force this
-  //   method to synchronously parse the buffer into a tree, even if it
-  //   otherwise would not. Defaults to `false`.
-  // * `options.comparisonRow`: A {Number} specifying the row to use as a
-  //   reference row. Must be a valid row that occurs earlier in the buffer
-  //   than `row`. If omitted, this method will determine the reference row on
-  //   its own.
+  // * `options` An optional {Object} will the following properties, all of
```

Comment:
```suggestion
  // * `options` An optional {Object} with the following properties, all of
```

Typo?

(from a while ago, but the indentation change put it in th diff, and I noticed it.)